libflux/src/semantic/flatbuffers/tests.rs

```diff
@@ -0,0 +1,67 @@
+extern crate flatbuffers; 
+
+use super::fbsemantic::fbsemantic::*; 
+
+#[test]
+fn test_flatbuffers_semantic() {
+    let mut builder = flatbuffers::FlatBufferBuilder::new_with_capacity(256);
+
+    // lets test out a unary expression using a float
+    let floatval = FloatLiteral::create(
+        &mut builder,
+        &FloatLiteralArgs {
+            value: 3.5,
+            ..FloatLiteralArgs::default()
+        },
+    );
+
+    let increment = UnaryExpression::create(
+        &mut builder,
+        &UnaryExpressionArgs {
+            operator: Operator::AdditionOperator,
```

Comment:
You know, it turns out that Flux does not have unary plus.  I just tried it out, I got an error.  Unary minus is a thing in Flux though.

(Unary plus definitely exists in C)

This test is still valid for your purposes though.

Reply:
Oh, that's good to know.

Reply:
Do you think I should change it to something that might actually show up in Flux?

Reply:
It's probably a good idea to change it, since it might confuse someone into thinking we have unary plus in Flux.

Reply:
💯 will do